bucket/mockery.json

```diff
@@ -0,0 +1,6 @@
+{
+  "version": "1.1.0",
+  "url": "https://github.com/vektra/mockery/releases/download/v1.1.0/mockery_1.1.0_Windows_x86_64.tar.gz",
+  "hash": "5df46b1b88b9f43d783cafe16d1cb49c753c4da17d43f482023b97e3abeb56ef",
+  "bin": "mockery.exe"
```

Comment:
```suggestion
    "version": "2.0.0-alpha.2",
    "desciption": "A mock code autogenerator for Golang",
    "homepage": "https://github.com/vektra/mockery",
    "architecture": {
        "64bit": {
            "url": "https://github.com/vektra/mockery/releases/download/v2.0.0-alpha.2/mockery_2.0.0-alpha.2_Windows_x86_64.tar.gz",
            "hash": "1a4af14d2a196b635192af33406e33cb4fd293eadcc9c64b41fed772560cd2ce"
        }
    },
    "bin": "mockery.exe",
    "checkver": {
        "github": "https://github.com/vektra/mockery",
        "regex": "tag/v([\\w.-]+)"
    },
    "autoupdate": {
        "architecture": {
            "64bit": {
                "url": "https://github.com/vektra/mockery/releases/download/v$version/mockery_$version_Windows_x86_64.tar.gz"
            }
        },
        "hash": {
            "url": "$baseurl/checksum.txt"
        }
    }
```

Reply:
Should we really publish an alpha version in scoop? I'd rather stick with the release version, or wait for the full release of 2.0.0

Reply:
bump @Ash258

Reply:
hello @Ash258